README.md

```diff
@@ -23,44 +23,49 @@ new [Vue Composition API](https://v3.vuejs.org/guide/composition-api-introductio
 ```typescript
 import { createApp } from 'vue';
 import CasbinPlugin from 'vue-authz';
-import { Authorizer } from 'casbin.js';
-
-const permission = {
-    "read": ['data1', 'data2'],
-    "write": ['data1']
-}
-
-// Run casbin.js in manual mode, which requires you to set the permission manually.
-const authorizer = new casbinjs.Authorizer("manual");
-
-authorizer.setPermission(permission);
+import { newEnforcer, MemoryAdapter } from 'casbin.js';
+
+const a = new MemoryAdapter("p, alice, data1, read\n" +
+    "p, alice, data2, read\n" +
+    "p, alice, data2, write\n" +
+    "p, bob, data2, write")
+
+enforcer = await newEnforcer("[request_definition]\n" +
+    "r = sub, obj, act\n" +
+    "\n" +
+    "[policy_definition]\n" +
+    "p = sub, obj, act\n" +
+    "\n" +
+    "[policy_effect]\n" +
+    "e = some(where (p.eft == allow))\n" +
+    "\n" +
+    "[matchers]\n" +
+    "m = r.sub == p.sub && r.obj == p.obj && r.act == p.act");
+
+const enforce = new casbinjs.enforcer.enforce("alice", "data1", "read");
 
 createApp()
-    .use(CasbinPlugin, authorizer, {
+    .use(CasbinPlugin, enforce, {
         useGlobalProperties: true
     }).mount('#app');
 ```
 
-After that, you can use `$authorizer` and `$can` in every component.
+After that, you can use `$enforce` in every component.
 
 ```vue
 
 <template>
-    <p
-        v-if='$can("read","post")'
-    >
+    <p v-if="$whatyouwant.enforce('alice', 'data2', 'write')">
```

Comment:
What is $whatyouwant?

Reply:
It was orignial designed to allow user define their own injection hook with `INJECT_KEY` to prevent namespace conflict. But correspond code was removed in this PR.